src/main/java/com/flansmod/common/driveables/EntitySeat.java

```diff
@@ -459,14 +462,30 @@ public void onMouseMoved(int deltaX, int deltaY)
 
 			float newPitch = 0f;
 			
+			
+			//Pitches the gun at the last possible moment in order to reach target pitch at the same time as target yaw.
+			float minYawToMove = 0f;
+			
+			float currentYawToMove = 0f;
+			
+			if(seatInfo.latePitch){
```

Comment:
I try to go for the open bracket on a new line coding standard. While this is my personal preference, and you may disagree, it really helps to have it consistent across the codebase. So I don't mind for this PR, but if I come across it myself while coding in this area, I will most likely reformat it as I go.